scripts/linters/custom_eslint_checks/lib/rules/break-after-parens.js

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Lint check to ensure that there is a break after parenthesis
+ * in case of multiline hanging indentation.
+ */
+'use strict';
+
+module.exports = {
+  meta: {
+    type: 'layout',
+    docs: {
+      description: 'There should be a break after parenthesis',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      expectedAfter: 'Expected newline after \'(\'.'
+    }
+  },
+
+  create: function(context) {
+    const sourceCode = context.getSourceCode();
+    const lines = sourceCode.lines;
+
+    return {
+      Program: function checkParenSpaces(node) {
+        const tokens = sourceCode.tokensAndComments;
```

Comment:
We are again using tokens for non-comment stuff! I see we are mostly using it for 
- function calls and expressions
- assignment operator
- list and dict.

Why aren't we doing it through nodes? The tokens are later read as a string/regex which looks bad!

Reply:
```
someFunction(parame1, param2, [
   <list of items>],
   Param4, param5)
```

@seanlip I and @DubeySandeep are having a confusion about this case, so just want to confirm with you if the above example is a valid case or an invalid case for break after a bracket.

Also, please consider if there is a `(` or `{` after 2nd argument, if that is a valid case or not?

Reply:
```
someFunction(parame1, param2, [
   <list of items>
], Param4, param5)
```

would be the preferred approach. The idea is to have the indentation level match for both "[" and "]".

The same holds for ( and {. In general there is no difference between how we treat the three types of "brackets".

Does this answer your question?

Reply:
@seanlip Yes, It answers the question, thanks for the confirmation.